dev/Common/CornerRadiusFilterConverters.idl

```diff
@@ -22,7 +26,9 @@ enum CornerRadiusFilterKind
     Top,
     Right,
     Bottom,
-    Left
+    Left,
+    TopLeft,
+    BottomRight
 };
```

Comment:
I was thinking the enum value itself could encode this. Like an enum named "TopLeftValue" would mean that I want a double back and not a CornerRadius.

Reply:
Hmm yeah, I think that's better, will update.